polymer/slambook/lib/tute_slambookform.dart

```diff
@@ -0,0 +1,158 @@
+// Copyright (c) 2012, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+@HtmlImport('tute_slambookform.html')
+library slambook.lib.tute_slambookform;
+
+import 'dart:html';
+import 'dart:convert';
+import 'package:polymer/polymer.dart';
+import 'package:web_components/web_components.dart' show HtmlImport;
+
+@PolymerRegister('tute-slambookform', extendsTag: 'form')
+class SlamBookComponent extends FormElement with 
+    PolymerMixin, PolymerBase, JsProxy {
+
+  AllAboutMe get myData => _myData;
+  AllAboutMe _myData = new AllAboutMe();
+
+  @Property(observer: 'submitForm')
+  void set myData(AllAboutMe newValue) {
+    _myData = newValue;
+    notifyPath('myData', _myData);
+  }
+
+  SlamBookComponent.created() : super.created() {
+    polymerCreated();
+  }
+
+  @override
+  void attached() {
+    super.attached();
+    notifyPath('myData', _myData);
```

Comment:
This shouldn't be necessary (which means you can probably scrap this whole method)